### PR TITLE
Unused code cleanup

### PR DESCRIPTION
**Pull request by arades79**
### Summary

Large purge of Onionshare code that is no longer used, as well as some cleanup in files that started from Onionshare.

All code was also re-formatted using black now, a future commit should add a pre-commit hook for black, and whenever CI/CD is worked out black -check will be included.

Some bandit lint notes were addressed as well to ensure the utmost in security. Namely all random operations were moved to use the secrets package for its cryptographic number generation.

No logic changes were made, and the app performs identically to how it did before, only with many unused code paths eliminated.

This partially addresses #12 which is an ongoing issue. Remaining items include changing icons, license files, and other meta-level vestige code to make the whole project bifurcated from Onionshare.
### Checklist

- [x] If code changes were made then they have been tested.
- [ ] This PR fixes an issue.
- [ ] This PR adds something new (e.g. new method or parameters).
- [ ] This PR is a breaking change (e.g. methods or parameters removed/renamed)
- [ ] This PR is **not** a code change (e.g. documentation, README, ...)

### CODE REVIEW

File: hyperdome_client/__init__.py [hunk 1]
```diff
@@ -38,15 +38,17 @@ class Application(QtWidgets.QApplication):
     """
 
     def __init__(self, common):
-        if common.platform == 'Linux' or common.platform == 'BSD':
+        if common.platform == "Linux" or common.platform == "BSD":
```
Comment (stevenpitts) [inline hyperdome_client/__init__.py:41]: You ignored the refactor suggestion? :o

Comment (arades79) [reply hyperdome_client/__init__.py:41]: I think there's enough suggestions for a wider-scale refactor and I didn't want to get too in the weeds on it

File: hyperdome_server/onion.py [hunk 2]
```diff
@@ -176,144 +188,177 @@ def __init__(self, common):
...
-                            'torrc_template-meek_lite_azure')) as o:
+                elif self.settings.get("tor_bridges_use_meek_lite_azure"):
+                    f.write(
+                        "ClientTransportPlugin meek_lite exec {}\n".format(
+                            self.obfs4proxy_file_path
+                        )
+                    )
+                    with open(
+                        self.common.get_resource_path("torrc_template-meek_lite_azure")
+                    ) as o:
```
Comment (stevenpitts) [inline hyperdome_server/onion.py:305]: I love these hanging indents, I didn't know you could close them like this (on the same column as the opening part)

Especially cuz it resolves the problem of being unable to have a hanging indent before an indented block

File: hyperdome_server/onion.py [hunk 3]
```diff
@@ -509,71 +573,77 @@ def start_onion_service(self, port):
...
 
         # Cleanup the ephemeral onion services, if we have any
         if self.c:
             onions = self.c.list_ephemeral_hidden_services()
             for onion in onions:
                 try:
-                    self.common.log('Onion', 'cleanup',
-                                    'trying to remove onion {}'.format(onion))
+                    self.common.log(
+                        "Onion", "cleanup", "trying to remove onion {}".format(onion)
```
Comment (stevenpitts) [inline hyperdome_server/onion.py:638]: slightly weird for this to be formatted differently than line 642

File: hyperdome_server/settings.py [hunk 4]
```diff
@@ -46,17 +45,20 @@ def __init__(self, common,
...
+                )
 
         # Dictionary of available languages in this version of OnionShare,
         # mapped to the language name, in that language
         self.available_locales = {
             # 'bn': 'বাংলা',       # Bengali
             # 'ca': 'Català',     # Catalan
             # 'da': 'Dansk',      # Danish
-            'en': 'English',    # English
+            "en": "English",  # English
```
Comment (stevenpitts) [inline hyperdome_server/settings.py:61]: Comment should probably match indenting of other comments around it

File: hyperdome_server/web.py [hunk 5]
```diff
@@ -0,0 +1,407 @@
...
+        """
+        Add security headers to a request
+        """
+        for header, value in self.security_headers:
+            r.headers.set(header, value)
+        return r
+
+    def _safe_select_jinja_autoescape(self, filename):
+        return filename is None or filename.endswith(
+            (".html", ".htm", ".xml", ".xhtml")
```
Comment (stevenpitts) [inline hyperdome_server/web.py:297]: I didn't know endswith could do multiple options, I can use that a lot :o

**Review by stevenpitts — APPROVED** [review]

File: hyperdome_server/onion.py (hunk 2 above)
Comment (arades79) [reply hyperdome_server/onion.py:305]: yeah it's pretty clean

File: hyperdome_server/onion.py (hunk 3 above)
Comment (arades79) [reply hyperdome_server/onion.py:638]: black formats according to the set line length, if it fits in the line it fits, only if it doesn't fit will it split it up. The philosophy of using black is to not argue with it, so I'm not arguing with it even though it's a little silly.

File: hyperdome_server/web.py (hunk 5 above)
Comment (arades79) [reply hyperdome_server/web.py:297]: huh, they don't even really show it in documentation either, it's just casually mentioned "oh you can use a tuple too" without really noting the implications.

**Pull request merged**